analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -120,13 +120,24 @@ class Gradle(
     private val dependencyHandler = GradleDependencyHandler(managerName, maven)
     private val graphBuilder = DependencyGraphBuilder(dependencyHandler)
 
+    // The path to the root project. In a single-project, just points to the project path. Is only null when not yet
```

Comment:
commit message: This might fix also the issue when you call the analyzer from outside the project dir and the build script uses git commands, assuming the working dir is inside the clone of the git repo. Parsing the results of git commands can have unexpected input and make the build script fail. 

Do you think that's worth mentioning?

Reply:
> the build script uses git commands

Yes, this is indeed also fixed by this change. I've generalized the commit message a bit in that regard.